config/robot.urdf

```diff
@@ -10,7 +14,7 @@
     </visual>
     <collision>
       <origin rpy="0 0 0" xyz="0 0 0.017"/>
-      <geometry>
+      <geometry>g
```

Comment:
Looks like some extra characters were accidentally introduced in the latest commit
```suggestion
      <geometry>
```